src/PJ_ccon.c

```diff
@@ -0,0 +1,94 @@
+/*
+
+	Central Conic Projection (spherical only, tangent, gnomonic)
+	
+	Very rarely used, implemented for ATPOL Polish Geobotanical Grid
+
+	see http://atpol.sourceforge.io
+
+	Lukasz Komsta, 2017
+
+*/
+
+
+#define PJ_LIB__
+#include <errno.h>
+#include <proj.h>
+#include "projects.h"
+
+struct pj_opaque {
+    double phi1;
+    double ctgphi1;
+    double sinphi1;
+    double cosphi1;
+    double *en;	
+};
+
+PROJ_HEAD(ccon, "Central Conic")
+    "\n\tCentral Conic, Sph.\n\tlat_1=";
+
+
+
+static XY e_forward (LP lp, PJ *P) {          
```

Comment:
The `e_` is usually used to specify that the function is working in the ellipsoidal case. Since you working on a spherical earth you should remove the `e_` to avoid confusion. The same goes for the inverse function below.